content/applications/inventory_and_mrp/manufacturing/advanced_configuration/sub_assemblies.rst

```diff
@@ -1,75 +1,76 @@
+Sublevel |BOMs| (subassemblies or semifinished products) streamline these production workflows, and
+are beneficial when the subassembly is used across multiple finished products (meaning they'd appear
+on multiple top-level |BOMs|). The more complicated a product is to build or procure, the more value
+a multilevel |BOM| can provide.
 
-.. seealso::
-   :doc:`../basic_setup/bill_configuration`
+Create a multilevel BoM
+=======================
+
+To set up a multilevel |BOM|, the top-level product and sublevel products's |BOMs| must be created.
+If starting from scratch, build the |BOMs| from the bottom up. Start with the lowest-level product
+|BOMs|, then include those products as components in higher-level |BOMs|.
+
+For example, a printed circuit board (PCB) for a custom keyboard would compose of hundreds
+electronic components, such as transistors, resistors, and capacitors. Instead of listing all of
+those components out, a sublevel product and |BOM| for `PCB` is created, to track the quantities of
+transistors and other small components, without needing to overcrowd the toplevel |BOM| for custom
+keyboard with listing them. Instead, the custom keyboard's |BOM| consists of an assortment of
+components and sublevel |BOMs| alike, like keycaps, switches, the PCB, and keyboard plate.
 
-.. image:: sub_assemblies/semifinished-product-bom.png
-   :align: center
-   :alt: A bill of materials for a semi-finished product.
+:doc:`Learn how to build a simple bill of materials <../basic_setup/bill_configuration>`. For the
+PCB, this would include the transistors, resistors, and other components.
 
-Create the top-level bill of materials (BoM)
-============================================
+.. image:: sub_assemblies/sublevel-bom.png
+   :alt: A bill of materials for a PCB.
 
-After the semi-finished products are fully configured, navigate to :menuselection:`Manufacturing
---> Products --> Products`. Then, :guilabel:`Create` the top-level product. Configure the product's
-specifications as desired, and be sure to :guilabel:`Save`.
+After the sublevel products (like the PCB, keycaps, and keyboard plate) are fully configured, create
+the top-level product by navigating to :menuselection:`Manufacturing --> Products --> Products`, and
+then selecting :guilabel:`New`. From here, configure the product's specifications as needed.
 
-Once the top-level product is configured, click the :guilabel:`Bill of Materials` smart button on
-the product form, then click :guilabel:`Create` to make a :abbr:`BoM (Bill of Materials)` for the
-top-level product. Then, simply add the semi-finished products to this :abbr:`BoM (Bill of
-Materials)`, along with any other necessary components.
+Once the top-level product (the keyboard) is configured, select the :guilabel:`Bill of Materials`
+smart button on the product form, then select :guilabel:`New` to make a |BOM| for the top-level
+product. Add the sublevel products to this |BOM|, along with any other necessary components.
```

Comment:
_Click_ is more natural choice for UI interactions involving buttons.
_Select_ is more appropriate for multiple options (like on a web form) and menus.

Adding the missing "and" conjunction makes the sentence grammatically sound.
```suggestion
Once the top-level product (the keyboard) is configured, click the :guilabel:`Bill of Materials`
smart button on the product form, and then select :guilabel:`New` to make a |BOM| for the top-level
product. Add the sublevel products to this |BOM|, along with any other necessary components.
```